lib/midcom/admin/folder/locale/default.de.txt

```diff
@@ -2,222 +2,430 @@
 ---VERSION 2.1.0
```

Comment:
This is the wrong file, it has nothing to do with invoices

Reply:
i changed this file to change the text on the button. it seems to work.

Reply:
all the l10n strings in the folder component have disappeared:

![Screenshot_20240320_115834](https://github.com/flack/openpsa/assets/425166/94e0f341-a3d1-4043-a807-9b61ae431997)